src/Analyzer.Core.UnitTests/TemplateAnalyzerTests.cs

```diff
@@ -58,6 +58,31 @@ public void AnalyzeTemplate_ValidInputValues_ReturnCorrectEvaluations(string res
             }
         }
 
+        [TestMethod]
+        public void AnalyzeTemplate_NotUsingPowerShell_NoPowerShellViolations()
```

Comment:
Maybe it's worth adding a test with the same template where there is a failure because TTK checks did run?

Reply:
Yes, that test is the last `DataRow` of the above test case.

Reply:
I was thinking of the case where we make changes such that that template doesn't trigger errors anymore. We will realize that we need to update the above test case, but we might forget that it's affecting this one too

Reply:
That's fair.  I will add the case in this same test method to ensure it's obvious what needs to change in the event of a test failure.